uiowa/uiowa
# Specify dependency on uids_base instead of uids for Yarn workspaces.

Resolves #3014 

# How to test

Follow the steps in #3014 to replicate the issue, assuming you don't mind nuking node_modules. Then do it again but make a manual change to the uiowa_apr/package.json file beforehand: replace the dependency on uiowa/uids with what it is changed to in this PR: "uids_base": "^1.0.0".

Then when you run the commands in #3014 you should see `=> Found "@uiowa/uids@3.1.10"` instead.

Other changes
- Update README docs on workspaces.
- Change APR SASS imports to use uids_base.
- Output uiowa/uids version after running `blt frontend`.
  - https://travis-ci.com/github/uiowa/uiowa/builds/215969158#L3643
